plugins/org.jkiss.dbeaver.ext.greenplum/plugin.xml

```diff
@@ -73,6 +81,40 @@
                     <property name="connectTimeout" value="20"/>
                     <property name="prepareThreshold" value="0"/>
                 </driver>
+                <driver
+                        id="cloudberry-jdbc"
+                        label="Cloudberry"
+                        icon="icons/cloudberry_icon.png"
+                        iconBig="icons/cloudberry_icon_big.png"
+                        class="org.postgresql.Driver"
+                        sampleURL="jdbc:postgresql://{host}[:{port}]/[{database}]"
+                        defaultPort="5432"
+                        webURL="https://cloudberrydb.org/"
+                        propertiesURL="https://jdbc.postgresql.org/documentation/head/connect.html#connection-parameters"
+                        databaseDocumentationSuffixURL="Database-driver-Greenplum"
+                        description="%driver.cloudberry.description"
+                        categories="sql,analytic">
+                    <replace provider="postgresql" driver="postgres-cloudberry-jdbc"/>
+
+                    <file type="jar" path="maven:/org.postgresql:postgresql:RELEASE[42.7.2]"
+                          bundle="!drivers.postgresql"/>
+                    <file type="jar" path="maven:/net.postgis:postgis-jdbc:RELEASE[2.5.0]" ignore-dependencies="true"
+                          optional="true" bundle="!drivers.postgresql"/>
+                    <file type="jar" path="maven:/net.postgis:postgis-jdbc-jtsparser:RELEASE[2.5.0]"
+                          ignore-dependencies="true" optional="true" bundle="!drivers.postgresql"/>
+                    <file type="jar" path="maven:/net.postgis:postgis-geometry:RELEASE[2.5.0]"
+                          ignore-dependencies="true" optional="true" bundle="!drivers.postgresql"/>
+                    <file type="license" path="https://raw.githubusercontent.com/pgjdbc/pgjdbc/master/LICENSE" bundle="!drivers.postgresql"/>
+
+                    <file type="license" path="drivers/postgresql/LICENSE.txt" bundle="drivers.postgresql"/>
+                    <file type="jar" path="drivers/postgresql" bundle="drivers.postgresql"/>
+                    <file type="jar" path="drivers/postgis" bundle="drivers.postgresql"/>
+                    <!--                    <file type="jar" path="drivers/waffle" bundle="drivers.postgresql"/>-->
```

Comment:
Let's not add commented-out code as we won't be able to identify its purpose in the future.